### PR TITLE
fix(core): optimize sidebar tag performance

### PR DESCRIPTION
**Pull request by EYHN**

### CODE REVIEW

**Comment by EYHN** [comment]
* **#7633** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7633?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#7632** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7632?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#7630** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7630?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @EYHN and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by pengx17 — APPROVED** [review]

**Review by forehalo — APPROVED** [review]

File: packages/frontend/core/src/modules/explorer/views/nodes/tag/index.tsx [hunk 1]
```diff
@@ -181,16 +181,27 @@ export const ExplorerTagNode = ({
...
     </ExplorerTreeNode>
   );
 };
+
+/**
+ * the `tag.pageIds$` has a performance issue,
+ * so we split the tag node children into a separate component,
+ * so it won't be rendered when the tag node is collapsed.
+ */
+export const ExplorerTagNodeDocs = ({ tag }: { tag: Tag }) => {
```
Comment (forehalo) [inline packages/frontend/core/src/modules/explorer/views/nodes/tag/index.tsx:194]: do we really need to show tagged docs in sidebar?

Comment (EYHN) [reply packages/frontend/core/src/modules/explorer/views/nodes/tag/index.tsx:194]: Of course, its in design

Comment (forehalo) [reply packages/frontend/core/src/modules/explorer/views/nodes/tag/index.tsx:194]: shit design

**Review by CatsJuice — APPROVED** [review]

**Pull request merged**